src/main/pages/che-7/installation-guide/proc_upgrading-che-using-the-cli-management-tool.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// upgrading-{prod-id-short}
+
+[id="upgrading-{prod-id-short}-using-the-cli-management-tool_{context}"]
+= Upgrading {prod-short} using the CLI management tool
+
+This section describes how to upgrade from previous minor version using the CLI management tool.
+
+.Prerequisites
+
+* And administrative account on an OpenShift instance.
+
+* A running instance of a previous minor version of {prod}, installed using the CLI management tool on the same instance of OpenShift.
+
+* An installation of the `{prod-cli}` management tool. See link:{site-baseurl}che-7/using-the-{prod-cli}-management-tool/[Using the `{prod-cli}` management tool].
```

Comment:
I don't think passive voice in these sentences has any negative impact. IMO, "X is installed" is easier to read/understand than "installation of X"

Reply:
Mainly to be consistent with https://www.eclipse.org/che/docs/che-7/installing-che-on-openshift-3-using-the-operator/